train.py

```diff
@@ -90,11 +90,11 @@ def train_image_bounding_box(
     default="https://cloud.kili-technology.com/api/label/v2/graphql",
     help="Kili Endpoint",
 )
-@click.option("--api-key", default=os.environ.get("KILI_API_KEY"), help="Kili API Key")
+@click.option("--api-key", default=os.environ.get("KILI_API_KEY", ""), help="Kili API Key")
 @click.option("--model-framework", default=None, help="Model framework (eg. pytorch, tensorflow)")
 @click.option("--model-name", default=None, help="Model name (eg. bert-base-cased)")
 @click.option("--model-repository", default=None, help="Model repository (eg. huggingface)")
-@click.option("--project-id", default=None, help="Kili project ID")
+@click.option("--project-id", default="", help="Kili project ID")
```

Comment:
Shouldn't we delete completely the default ?

Reply:
adding `required=True` instead